webui/react/src/components/GridListRadioGroup.tsx

```diff
@@ -24,15 +24,17 @@ const GridListRadioGroup: React.FC<Props> = ({ onChange, value }: Props) => {
   );
 
   return (
-    <RadioGroup
-      iconOnly
-      options={[
-        { icon: 'grid', id: GridListView.Grid, label: 'Grid View' },
-        { icon: 'list', id: GridListView.List, label: 'List View' },
-      ]}
-      value={value}
-      onChange={handleChange}
-    />
+    <div data-testid="grid-list-radio-group">
```

Comment:
in general, i prefer data-test-component as a top level testid in case another `data-testid` get added when `RadioGroup` is implemented. nbd either way